lib/puppet/module.rb

```diff
@@ -60,6 +60,10 @@ def self.is_module_namespaced_name?(name)
 
   def initialize(name, path, environment)
     @name = name
+    # TODO: testing to see if necessary to wrap path with expand_path
+    not_expanded = (path.index('~'))
```

Comment:
Remove redundant parenthesis. Ideally, since `not_expanded` is used as a boolean, coerce it. I.e.
```
not_expanded = !!path.index('~')
```